src/ngraph/op/convolution.cpp

```diff
@@ -364,18 +323,32 @@ void op::ConvolutionBackpropData::generate_adjoints(autodiff::Adjoints& adjoints
     CoordinateDiff padding_below;
     CoordinateDiff padding_above;
     Strides data_dilation_strides;
+    const Shape& filters_shape = get_input_shape(0);
     for (size_t i = 0; i < f_shape.size() - 2; i++)
     {
-        window_movement_strides.push_back(m_window_dilation_strides_backward[i]);
-        window_dilation_strides.push_back(m_window_movement_strides_backward[i]);
-        padding_below.push_back(m_padding_below_backward[i]);
-        padding_above.push_back(m_padding_above_backward[i] -
-                                (m_padding_below_backward[i] +
-                                 (x_shape[i + 2] - 1) * m_data_dilation_strides_backward[i] +
-                                 m_padding_above_backward[i] -
-                                 (f_shape[i + 2] - 1) * m_window_dilation_strides_backward[i]) %
-                                    m_window_movement_strides_backward[i]);
-        data_dilation_strides.push_back(m_data_dilation_strides_backward[i]);
+        window_movement_strides.push_back(m_window_dilation_strides_forward[i]);
```

Comment:
`m_window_dilation_strides_forward[i]` here was originally
`m_window_dilation_strides_backward[i]` which was defined as
`m_window_dilation_strides_forward[i]`.

So this looks good.